### DISCUSSION TITLE
Output format of Neper

### DISCUSSION THREAD

**Discussion by Dreamown** [neperfepx/neper#400] [Q&A] [answered]
Is there a feasible way to make the model generated by Neper open in CAD format, such as Stp,step,iges, instead of OBJ?

**Comment by rquey** [comment, marked as answer]
These formats are not available in Neper, and I don't know them, but they could probably be added.

**Reply by Dreamown** [reply]
Thanks for your reply, I seem to have found a way.

**Reply by ap264** [reply]
Hi Dreamown

May I know how to convert the tess file to CAD format?

Thank you

**Reply by Dreamown** [reply]
You can write a Python script that reads point, line and surface information in Tess or OBj format. You can even write a tool with Grasshopper parameter modeling platform, which is very convenient.

**Reply by ap264** [reply]
Hi Dreamown 

Thank you for the reply. I would like to say that I am not that well versed with python. But I will do my research and try to understand how to write the script. I would like to know if the python script alone is enough to get my tessellation in CAD format. Also, I would like to know if it is possible for you to share your or any other python script material from which I could understand how to write this script. 

Also, Is the grasshopper software is a plugin for rhino?  Should l download rhino and then run the grasshopper?
